Arrowgene.Ddon.Shared/Entity/PacketStructure/S2CCraftStartEquipColorChangeRes.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Entity.Structure;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure
+{
+    public class S2CCraftStartEquipColorChangeRes : ServerResponse
+    {
+        public override PacketId Id => PacketId.S2C_CRAFT_START_EQUIP_COLOR_CHANGE_RES;
+
+        public S2CCraftStartEquipColorChangeRes()
+        {
+            CurrentEquipInfo = new CDataCurrentEquipInfo();
+        }
+
+        public byte ColorNo { get; set; } // Not sure why its called this in the PS4 Symbols, just rolling with it for now until i know how this works.
+        public CDataCurrentEquipInfo CurrentEquipInfo { get; set; }
+
+        public class Serializer : PacketEntitySerializer<S2CCraftStartEquipColorChangeRes>
+        {
+            public override void Write(IBuffer buffer, S2CCraftStartEquipColorChangeRes obj)
+            {
+                WriteByte(buffer, obj.ColorNo);
```

Comment:
First line in this `Write` function needs to be `WriteServerResponse(buffer, obj);`